nexus/external-api/output/nexus_tags.txt

```diff
@@ -272,6 +272,11 @@ local_idp_user_delete                    DELETE   /v1/system/identity-providers/
 local_idp_user_set_password              POST     /v1/system/identity-providers/local/users/{user_id}/set-password
 saml_identity_provider_create            POST     /v1/system/identity-providers/saml
 saml_identity_provider_view              GET      /v1/system/identity-providers/saml/{provider}
+scim_idp_create_token                    POST     /v1/system/identity-providers/scim/tokens
+scim_idp_delete_all_tokens               DELETE   /v1/system/identity-providers/scim/tokens
+scim_idp_delete_token_by_id              DELETE   /v1/system/identity-providers/scim/tokens/{token_id}
+scim_idp_get_token_by_id                 GET      /v1/system/identity-providers/scim/tokens/{token_id}
+scim_idp_get_tokens                      GET      /v1/system/identity-providers/scim/tokens
```

Comment:
Ok, I see that a query param is used to select the silo in all these cases. I assume there can only be one SAML IdP for a given silo, and that's why there doesn't need to be a `/scim/{provider}/tokens`?

Reply:
If it doesn't hang off an identity provider and is 1-1 with a silo, I think that feels better. Do you expect to add more endpoints under `/v1/system/scim` besides `/tokens`?

Reply:
I was expecting this to sit within the silo access layout, but it sounds like based on this endpoint it'd be in the the system silo page instead, and would require fleet admin perms?

<img width="1039" height="566" alt="image" src="https://github.com/user-attachments/assets/2e880315-e515-4ecb-8871-2ea67bdf976f" />

Reply:
@benjaminleonard SCIM client token CRUD requires silo admin permissions, not fleet - it'll be the same permission requirement as the rest of the IDP CRUD.

Reply:
And to answer the original question, I think this means that it makes the most sense to stick it in the system side of the console, though that means a user that is only a silo admin and not at least a fleet viewer would not be able to get to it. I suppose the most robust thing would be to have it show up both on the system side and silo side, but that sounds terrible. Like I said, we might need to think about this system/silo boundary a bit more. Maybe there should be pages that are kind of in neither space but can be _gotten to_ from either one. That feels kinda bad too, though the existing pages for user settings and SSH keys are kind of like that.